ArticleService.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+
+const URL = 'https://panda-market-api-crud.vercel.app/articles/';
+
+export async function getArticleList({page, pageSize, keyword}) {
+  return fetch(URL,{
+      params: {page, pageSize, keyword}
+    })
+  .then(res => res.json())
+  .then(data => {
+    return data
+  })
+  .catch((e) => {
+    if(!res.ok) {
+    console.log('목록을 가져오는 중 오류가 발생했습니다.');
+  }
+});
+}
+
+export async function getArticle(articleId) {
+  const res = await axios.get(`${URL}${articleId}`)

```

Comment:
URL에 `/`가 빠진 것 같습니다!

```suggestion
  const res = await axios.get(`${URL}/${articleId}`)

```